client/src/rr/dnssec/key_format.rs

```diff
@@ -18,6 +22,8 @@ pub enum KeyFormat {
     Pem,
     /// Raw bytes unformatted
     Raw,
```

Comment:
No. I plan on deleting Raw. 

Do you plan on supporting password protected pkcs8, btw?